src/libtest/stats.rs

```diff
@@ -11,7 +11,7 @@
 #![allow(missing_docs)]
 #![allow(deprecated)] // Float
 
-use std::cmp::Ordering::{self, Less, Greater, Equal};
+use std::cmp::Ordering::{self, Equal, Greater, Less};
```

Comment:
Worth filing an issue against rustfmt about this. My personal experience is that in 99% of cases, programmers don't order their imports and it's not worth the extra effort for the other 1%. In particular I'm not sure if it helps anybody - I agree the enum ordering is sensible, but I don't see any concrete benefit.